db.go

```diff
@@ -33,7 +33,7 @@ type DB interface {
 	SetConnMaxLifetime(d time.Duration)
 	SetMaxIdleConns(n int)
 	SetMaxOpenConns(n int)
-	// Stats() sql.DBStats // not relevant for multi connection DB
+	Stats() sql.DBStats // not relevant for multi connection DB
```

Comment:
please modify the comment, maybe "only for primary db" (?)

Reply:
Done